src/service/jank-meter.js

```diff
@@ -182,7 +182,10 @@ function isJankMeterEnabled(win) {
  * @returns {boolean}
  */
 function isLongTaskApiSupported(win) {
-  return !!win.PerformanceObserver
+  // TODO (@lannka, #9749) Turn performanceObserver on.
+  const tempOff = false;
```

Comment:
I am trying to keep the old so it would convenient to turn it on later. But sure, I am good with `returning false`